google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java

```diff
@@ -177,6 +180,21 @@ public static EntryListOption sortOrder(SortingField field, SortingOrder order)
     public static EntryListOption filter(String filter) {
       return new EntryListOption(OptionType.FILTER, filter);
     }
+
+    /** Returns an option to specify a organization to the log entries to be listed. */
```

Comment:
a organization to --> an organization for

Reply:
Done.